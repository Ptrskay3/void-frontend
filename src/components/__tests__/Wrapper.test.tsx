import React from "react";
import { shallow } from "enzyme";
import { Wrapper } from "../Wrapper";

describe("Layout", () => {
  it("Basic small Wrapper renders", () => {
    const layout = shallow(<Wrapper variant="small" />);
    expect(layout.getElements()).toMatchSnapshot();
  });

  it("Basic regular Wrapper renders", () => {
    const layout = shallow(<Wrapper variant="regular" />);
    expect(layout.getElements()).toMatchSnapshot();
  });
});
