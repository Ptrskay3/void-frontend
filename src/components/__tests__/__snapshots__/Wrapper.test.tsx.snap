// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout Basic regular Wrapper renders 1`] = `
Array [
  <Box
    maxW="800px"
    mt={8}
    mx="auto"
    width="100%"
  />,
]
`;

exports[`Layout Basic small Wrapper renders 1`] = `
Array [
  <Box
    maxW="400px"
    mt={8}
    mx="auto"
    width="100%"
  />,
]
`;
